NuRadioReco/framework/station.py

```diff
@@ -135,7 +135,24 @@ def get_channel_ids(self, return_group_ids=False):
 
         return list(channel_ids)
 
-    def add_channel(self, channel):
+    def add_channel(self, channel, overwrite=True):
```

Comment:
I am not sure I like overwrite = True as default ... that seems dangerous.

Reply:
Well I thought I keep the original behaviour as default behaviour to not break anything...

Reply:
Okay lets see what happens. I will divert all blame towards you :D